rust/lang/security/audit/dangerous-exec-command.yaml

```
@@ -0,0 +1,32 @@
+rules:
+  - id: dangerous-exec-command
+    patterns:
+      - pattern-either:
+          - patterns:
+              - pattern-either:
+                  - pattern: |
+                      Command::new($CMD)
```

```suggestion
          - pattern: |
                      Command::new($CMD)
```
A few unnecesary operators